doc/wiki/dashboard-audit.md

```diff
@@ -0,0 +1,668 @@
+WHERE "patients"."deleted_at" IS NULL
+  AND "patients"."registration_facility_id" = $1
+  AND "medical_histories"."deleted_at" IS NULL
+  AND "medical_histories"."hypertension" = $2
+  AND ("patients"."recorded_at" IS NOT NULL)
+GROUP BY date_trunc('month', "patients"."recorded_at"::timestamptz AT TIME ZONE 'ETC/UTC') AT TIME ZONE 'ETC/UTC' [["registration_facility_id", "2e768fb2-9de0-4a7d-a64d-9f5b4c1863f4"],["hypertension", "yes"]]"
+```
+
+**Where it's shown in the Dashboard** 
++ **Reports:** Overview, Details
++ **Home:** BP controlled, BP not controlled, Missed visits
++ **Simple app:** Progress tab
+
+**Things we should fix**
++ [ ] Rename query name from "Monthly registered patients" to "New registered patients"
++ [ ] Make copy more specific and more closely match how the query is calculated
++ [ ] Resolve query duplication where needed and ensure all queries return the right data
+
+---
+
+## Total registered patients
+**Copy used in the Dashboard**
+Total hypertension patients registered in `region_name`
+
+**How it's calculated**
+The sum of all monthly registrations at a facility
+
+**Where it's shown in the Dashbaord**
++ **Reports:** Overview, Details
++ **Home:** BP controlled, BP not controlled, Missed visits
++ **Simple App:** Progress tab
+
+**Things we should fix**
++ [ ] Make copy definition more specific and more closely match how the query is calculated
++ [ ] Resolve query duplication where needed and ensure all queries return the right data
+
+---
+
+## Total assigned patients
+**Copy used in the Dashboard**
+Hypertension patients assigned to `region_name`
+
+**How it's calcualted**
+The number of patients assigned to a facility where the patient:
++ Is not deleted
++ Is hypertensive
++ Is not dead
+
+*Sample SQL query*
+```sql
+SELECT COUNT(DISTINCT "patients"."id") AS count_id,
+       DATE_TRUNC('month', "patients"."recorded_at"::timestamptz AT TIME ZONE 'ASIA/KOLKATA') AT TIME ZONE 'ASIA/KOLKATA' AS date_trunc_month_patients_recorded_at_timestamptz_at_time_zone_
+FROM "patients"
+INNER JOIN "medical_histories" ON "medical_histories"."deleted_at" IS NULL
+AND "medical_histories"."patient_id" = "patients"."id"
+WHERE "patients"."deleted_at" IS NULL
+  AND "patients"."assigned_facility_id" = $1
+  AND "medical_histories"."deleted_at" IS NULL
+  AND "medical_histories"."hypertension" = $2
+  AND "patients"."status" != $3
+  AND ("patients"."recorded_at" IS NOT NULL)
+GROUP BY DATE_TRUNC('month', "patients"."recorded_at"::timestamptz AT TIME ZONE 'ASIA/KOLKATA') AT TIME ZONE 'ASIA/KOLKATA' [["assigned_facility_id", "fdbeb339-770f-4021-a81a-44f9bc9a5e79"],["hypertension", "yes"],["status", "dead"]]
+```
+
+**Where it's shown in the Dashboard**
++ **Reports:** Overview, Details, Cohort reports
++ **Home:** BP controlled, BP not controlled, Missed visits
++ **Simple App:** Progress tab
+
+**Things we should fix**
++ [ ] Make copy definition more specific and more closely match how the query is calculated
```

Comment:
I'm not sure I see the issue with the copy here.

Reply:
The issue is that the definition of a word can't include the word itself. Saying that "Assigned patients" is "All patients assigned to X" doesn't explain what "assigned" actually means.